packages/rancher-windows-exporter/charts/scripts/proxy-entry.ps1

```diff
@@ -0,0 +1,11 @@
+# default
+$listenPort = "9796"
```

Comment:
https://github.com/rancher/rancher/issues/28327#issuecomment-754274987 << basedd on this comment I am wondering if users will still have to keep this port reserved for metrics? or how does this setup accomplish that?

Reply:
Good question! The port on the host will still have to be reserved for `wmi_exporter`; we're only publishing the port on the host via the Pod. 

The difference though is that you will not have to open up the node port on the host to the external world (e.g. in Monitoring V1, --exposes would actually modify the firewall rules on the Windows host itself to allow traffic outside of `localhost` or `127.0.0.1` to send packets to that port). Additionally, you would not need to modify your Cloud Provider based firewall rules (e.g. security groups) to expose that port either.

Reply:
gotcha!! thank you for the explanation! 😄